addon/index.js

```diff
@@ -439,12 +439,12 @@ exports.main = function(options) {
     store.clientUUID = require('sdk/util/uuid').uuid().toString().slice(1, -1);
   }
 
+  Metrics.init();
```

Comment:
"this comment" doesn't seem to go anywhere, now... What was the feedback on this? Seems weird to start from time zero on every browser restart and upgrade of the main add-on